backend-server/shared/starters/oss/src/main/java/com/apitable/starter/oss/autoconfigure/OssProperties.java

```diff
@@ -38,6 +38,8 @@ public class OssProperties {
 
     private Aliyun aliyun;
 
+    private HuaweiCloud huaweicloud;
```

Comment:
the properties key `huaweicloud` can not match `huawei-cloud`, the  key name rules must be `lowerCamelCase`

Reply:
the key name has been `lowerCamelCase`